pubpub/pubpub
# Bh/light dark theme spubheader

Quick fix changes to make the recently included Submit Pub button match the rest of the header buttons through all light/dark theme selections.

_Test plan_

1. navigate to an incomplete pub on this PR's dyno
2. go to the preview tab
3. confirm the 'submit pub' button matches the styling of the other header buttons
4. in duqduq, find the same pub and change the light/dark text background in the theme
5. go back to this pr's dyno, and confirm the theming has changed

this will be easier to test after getting the 'edit theme' functionality added
